addons/editor-comment-previews/userstyle.css

```
@@ -16,8 +16,8 @@
   white-space: pre-wrap;
   pointer-events: none;
 
-  color: rgb(87, 94, 117);
-  background-color: rgb(255 255 255 / 90%);
+  color: var(--editorDarkMode-input-text, #575e75);
+  background-color: var(--editorDarkMode-input-transparent90, rgba(255, 255, 255, 0.9));
```

```suggestion
  background-color: var(--editorDarkMode-input-transparent90, rgb(255 255 255 / 0.9));
```
I think it's better to keep this syntax for colors because it's also used elsewhere in this file.